app/src/main/java/com/brainwallet/ui/composable/bottomsheet/FiatSelectorBottomSheet.kt

```diff
@@ -0,0 +1,58 @@
+package com.brainwallet.ui.composable.bottomsheet
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CheckCircle
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.ListItemDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import com.brainwallet.data.model.CurrencyEntity
+import com.brainwallet.tools.sqlite.CurrencyDataSource
+import com.brainwallet.ui.composable.BrainwalletBottomSheet
+import com.brainwallet.ui.theme.BrainwalletTheme
+
+/**
+ * describe [FiatSelectorBottomSheet] for CurrencySelector
```

Comment:
👨‍🎓  ✋ : Is there a shortcut to add the document comment @andhikayuana ?

Reply:
the content is manual, but for the comment related you can do
* Comment/uncomment with line comment: Command+/
* Comment/uncomment with block comment: Command+Shift+/

more: https://developer.android.com/studio/intro/keyboard-shortcuts